protobuf/state/state.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package statepb;
+
+option go_package = "github.com/NordSecurity/nordvpn-linux/state/pb";
+
+message Empty {}
+
+message AppState {
+    oneof state {
+        bool vpn_enabled = 1;
```

Comment:
I don't think this will be sufficient. This should be an enum with at least 3 states, optionally 4.
* connecting
* connected
* disconnecting (optional)
* disconnected
In the `connecting` and `connected` state it should also include the server(or the peer) info, maybe more info if needed.